gettsim/data/e_st_abzuege.yaml

```
@@ -51,6 +51,38 @@ kifreib:
     2019-01-01:
       value: 7620
       note:
+kifreib_s:
+  name:
+    de: Kinderfreibetrag (nur sächliches Existenzminimum).
+    en:
+  description:
+    de: § 32(6) EStG
+    en:
+  values:
+    2002-01-01:
+      value: 1824
+      note:
```

Could you add a note here? [Quick search](https://offenegesetze.de/veroeffentlichung/bgbl1/2001/42?q=%221%20824%22#page=2) reveals 
```suggestion
      note: Artikel 1 G. v. 16.08.2001 BGBl. I S. 2074.
```